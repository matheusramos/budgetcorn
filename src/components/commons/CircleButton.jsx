/* @flow */
import React from 'react';
import styled from 'styled-components';

const RoundButton = styled.button`
  border-radius: 100%;
`;
export default (props: { children: React.Node }) => {
  const { children, ...otherProps } = props;
  return <RoundButton {...otherProps}>{children}</RoundButton>;
};
