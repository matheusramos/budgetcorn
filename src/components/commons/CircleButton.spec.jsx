/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';

import CircleButton from './CircleButton';

it('renders correctly', () => {
  const tree = renderer.create(<CircleButton>Test</CircleButton>).toJSON();
  expect(tree).toMatchSnapshot();
});
