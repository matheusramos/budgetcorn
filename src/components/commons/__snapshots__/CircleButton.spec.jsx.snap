// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="sc-bdVaJa cruQxV"
>
  Test
</button>
`;
